Resources/Prototypes/_EE/Contractors/nationality.yml

```
@@ -97,3 +97,13 @@
     - !type:CharacterSpeciesRequirement
       species:
         - IPC
+
+- type: nationality
+  id: Qiimaeruan
+  nameKey: nationality_name_qiimaeruan
+  descriptionKey: nationality_description_qiimaeruan
+  passportPrototype: QiimaeruPassport
+  requirements:
+    - !type:CharacterSpeciesRequirement
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)